page/__tests__/functionality_unit.test.mjs

```diff
@@ -15,5 +15,139 @@ describe('Page endpoint functionality unit test (just testing helper functions).
     let page = await findPageById(123)
     expect(page).not.toBe(null)
   })
+  describe('Page endpoint functionality unit test (just testing helper functions). #functions_unit', () => {
+    it('No TPEN3 page id provided. Page validation must be false.', () => {
+      expect(validateID()).toBe(false);
+    });
+  
+    it('Detect TPEN3 page does not exist. The query for a TPEN3 page must be null.', async () => {
+      const page = await findPageById(-111);
```

Comment:
I mean, these tests are all written to pass explicitly so it was sort of a waste of time, but it can be cleaned up later.